mailet/standard/src/test/java/org/apache/james/transport/matchers/SMTPAuthSuccessfulTest.java

```diff
@@ -17,29 +17,53 @@
  * under the License.                                           *
  ****************************************************************/
 
-
 package org.apache.james.transport.matchers;
 
-import org.apache.mailet.base.GenericMatcher;
+import static org.assertj.core.api.Assertions.assertThat;
 
-public class SMTPAuthSuccessfulTest extends AbstractHasMailAttributeTest {
+import java.util.Collection;
 
-    protected String getHasMailAttribute() {
-        return "";
-    }
+import org.apache.mailet.MailAddress;
+import org.apache.mailet.base.test.FakeMail;
+import org.apache.mailet.base.test.FakeMailContext;
+import org.apache.mailet.base.test.FakeMatcherConfig;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SMTPAuthSuccessfulTest {
 
-    protected GenericMatcher createMatcher() {
-        return new SMTPAuthSuccessful();
+    private SMTPAuthSuccessful testee;
+
+    @Before
+    public void setUp() throws Exception {
+        testee = new SMTPAuthSuccessful();
+        testee.init(FakeMatcherConfig.builder().matcherName("matcherName")
+            .mailetContext(FakeMailContext.defaultContext())
+            .build());
     }
 
-    protected String getMatcherName() {
-        return "SMTPAuthSuccessful";
+    @Test
+    public void matchShouldReturnRecipientsWhenAuthUserAttributeIsPresent() throws Exception{
+        MailAddress recipient = new MailAddress("cuong.trn@gmail.com");
```

Comment:
Looks like. I modified some of the address they wrote by I guess I missed this one.